automol/util/matrix.py

```diff
@@ -7,9 +7,8 @@ def string(mat: Sequence[Sequence[float]], val_format="{0:>8.3f}") -> str:
     """Write a matrix to a string.
 
     :param mat: matrix to form string with
```

Comment:
For consistency, let's try to start all parameter descriptions with an upper case letter.